package.json

```diff
@@ -1,8 +1,11 @@
 {
   "name": "fastly-promise",
-  "version": "1.0.1",
+  "version": "2.0.0",
```

Comment:
I know it's a breaking change but does that warrant a full version change?  Not just 1.1.0?